### ISSUE TITLE
Chat2 changes live screen

### ISSUE THREAD

**Issue by unconservo** [WWBN/AVideo#4281]
Hi. When live streaming and opening chat on the stream display it moves camera to right. Then closing chat2 it leaves it like that. This is on android phone testing.

Also it takes up half the livestream screen. 

![20210101_090520](https://user-images.githubusercontent.com/76112286/103434875-03b03780-4c10-11eb-8190-1dd8a1e51341.jpg)
![20210101_090436](https://user-images.githubusercontent.com/76112286/103434877-06129180-4c10-11eb-992a-e9293e4aa3f6.jpg)

**Comment by DanielnetoDotCom** [comment]
This is not Chat 2

I guess you are using the deprecated Live Chat

**Comment by unconservo** [comment]
Hi. I activated both. To see differences. I then disabled live chat and
only kept chat 2 active. Chat 2 opens up in middle right of livestream and
cannot move it . It is also not transparent so you csnt see half the
stream. Is there settings i have to change?

Also whenever i start a live stream i have to change phone to landscape as
it brings up a block over live stream and can only see small thin line of
live stream under block.

Also local server shows offline but it is not.
